src/main/kotlin/minesweeper/domain/Mines.kt

```diff
@@ -0,0 +1,31 @@
+package minesweeper.domain
+
+class Mines(private val mines: List<Mine>) {
+
+    fun hasDuplicate(): Boolean {
+        val nonDuplicateMines = mines.toSet()
+        if (nonDuplicateMines.size != this.mines.size) return true
+        return false
```

Comment:
아래의 코드처럼 한 줄로 줄일 수 있어요.

```suggestion
        return nonDuplicateMines.size != this.mines.size
```